helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl

```diff
@@ -884,6 +1016,20 @@ service:
         {{- end }}
         {{- end }}
     {{- end }}
+
+    {{- if and (eq (include "splunk-otel-collector.splunkO11yEnabled" .) "true") .Values.featureGates.useControlPlaneMetricsHistogramData }}
+    metrics/histograms:
```

Comment:
I haven't noticed any filtering for the non-histogram metrics collected by the new Prometheus receivers. Are we using a default set of metrics or applying a filter somewhere? Without filtering, we might duplicate existing control plane metrics and inadvertently collect X number of new or custom control plane metrics.

Reply:
We should activate one or the other if you scroll up ; we will need to test for it.